app/controllers/recipes_controller.rb

```diff
@@ -1,9 +1,10 @@
 class RecipesController < ApplicationController
+
   def index
     @response = HTTParty.get("https://spoonacular-recipe-food-nutrition-v1.p.mashape.com/recipes/random?limitLicense=false&number=10", headers:{"X-Mashape-Key" => "lIOsRhZMTKmshyZ09B79rcDrwYNJp1gIzZmjsnPFBdpnDRnySc","Accept" => "application/json"})
     @recipes =@response["recipes"].each do |recipe|
```

Comment:
I changed this line to `@recipes =@response["recipes"].map do |recipe|`

notice I changed `#each` to `#map`.